gravitee-am-ui/src/app/domain/applications/application/overview/overview.component.ts

```diff
@@ -70,6 +70,10 @@ export class ApplicationOverviewComponent implements OnInit {
     return this.application.type.toLowerCase() === 'service';
   }
 
+  isUmaApp(): boolean {
+    return this.application.type.toLowerCase() === 'resource_server';
```

Comment:
The application should be considered as a resource application as soon as it get the `uma_protection` scope.  
By running current uma postman scenario, the current "RS-Client" is not considered as a UMA resource app while it can generate resource set & permissions tickets.  
Should we update the application type automatically after every scopes changes?

Reply:
For now I will leave it as it is.